### ISSUE TITLE
export csv file

### ISSUE THREAD

**Issue by picosmom2005** [eleybourn/Book-Catalogue#730]
I have a new phone and installed book catalogue I also moved my csv file to my sd card and the program still is not finding it please help I really have a lot of books and do not wish to reenter them

**Comment by eleybourn** [comment]
Have you followed the instructions in the wiki: https://github.com/eleybourn/Book-Catalogue/wiki/Frequently-Asked-Questions-(FAQ)

**Comment by picosmom2005** [comment]
yes I have and it does not find the one I transferred back from my computer

**Comment by eleybourn** [comment]
Could you go to the help page and select copy "send information". We'll have a look at this issue.

**Comment by Grunthos** [comment]
As a test, first run an export to CSV from the app, then move the csv file to 
the corrrect directory (ie. where the export was just created) then rename your 
file 'export.csv' (if you didn't already overwrite the one you created in the 
first step), then try an import. If that does not work, go to 'Help'->'Send 
Information'
